apis/cr/v1/doc.go

```diff
@@ -1,3 +1,76 @@
+/*
+Crunchy PostgreSQL Operator API
+
+The Crunchy PostgreSQL Operator API defines HTTP(S) interactions with the Crunchy PostgreSQL Operator.
+
+
+## Direct API Calls
+
+The API can also be accessed by interacting directly with the API server. This can be done by making HTTP requests with curl to get information from the server. In order to make these calls you will need to provide certificates along with your request using the `--cacert`, `--key`, and `--cert` flags. Next you will need to provide the username and password for the RBAC along with a header that includes the content type and the `--insecure` flag. These flags will be the same for all of your interactions with the API server and can be seen in the following examples.
+
+
+###### Get API Server Version
+
+The most basic example of this interaction is getting the version of the API server. You can send a GET request to `$PGO_APISERVER_URL/version` and this will send back a json response including the API server version. You must specify the client version that matches the API server version as part of the request.
+
+The API server is setup to work with the pgo command line interface so the parameters that are passed to the server can be found by looking at the related flags. For example, the series parameter used in the `create` example below is the same as the `-e, --series` flag that is described in the [pgo cli docs](https://access.crunchydata.com/documentation/postgres-operator/4.1.0/operatorcli/cli/pgo_create_cluster/).
+```
+curl --cacert $PGO_CA_CERT --key $PGO_CLIENT_KEY --cert $PGO_CA_CERT -u pgoadmin:examplepassword -H "Content-Type:application/json" --insecure -X GET $PGO_APISERVER_URL/version
+```
```

Comment:
I would recommend truncating the line length at 80 for this file.

For examples that involve shell commands, you can continue a command onto a newline with `/`